src/Uno.Toolkit.Skia.WinUI/Controls/Shadows/ShadowContainer.cs

```diff
@@ -256,9 +257,26 @@ protected override void OnApplyTemplate()
 	private void InvalidateCanvasLayout()
 	{
 		if (Content is not FrameworkElement contentAsFE ||
```

Comment:
dont align code, just indent them 1level per nesting
also even if you were to align them, it should be aligned to `if (`, not the `is not`